defines missing load balancer health check reference and allows load balancer private ip

makarono opened the pull request:
- defines missing load balancer health check reference
- allows load balancer private ip
- both changes are tested in production
- THANK YOU for this nice module!!

Commit:
defines missing load balancer health check reference and allows load balancer private ip

zoro16 commented:
Thanks for the great contribution!!

zoro16 reviewed (commented):
Looking good!!

There is an issue in the CI, will fix first before merging this PR.

zoro16 committed:
chore: bump pre-commit version - update terraform docs

zoro16 committed:
chore: bump hcloud provider version

zoro16 committed:
chore: bumped pre-commit-terraform version

zoro16 committed:
fix: load balancer algorthim type

zoro16 committed:
chore: updated readme docs

zoro16 committed:
chore(ci): bumped terraform-docs and tflint versions

zoro16 committed:
fix: set correct default values and fixed validation

zoro16 committed:
chore: updated docs

zoro16 committed:
fix(ci): added terraform set to CI

zoro16 committed:
fix(ci): fix yaml syntax

semantic-release-bot committed:
chore(release): version 1.0.1 [skip ci]

## [1.0.1](https://github.com/zoro16/terraform-hcloud-load-balancer/compare/v1.0.0...v1.0.1) (2025-01-23)

### Bug Fixes

* **ci:** added terraform set to CI ([2787501](https://github.com/zoro16/terraform-hcloud-load-balancer/commit/2787501fe284f297c264177e67df3b5f6b78a33e))
* **ci:** fix yaml syntax ([cf3d785](https://github.com/zoro16/terraform-hcloud-load-balancer/commit/cf3d7851e86b9dcb5fa1c8ce45bb495a6171bb3e))
* load balancer algorthim type ([c273282](https://github.com/zoro16/terraform-hcloud-load-balancer/commit/c273282cb30a2fa5a569b0a030c48c4111864fae))
* set correct default values and fixed validation ([9aa1a53](https://github.com/zoro16/terraform-hcloud-load-balancer/commit/9aa1a536f2662eb8e86b8ebc177982c3a1ce6608))

zoro16 commented:
@makarono I have pushed many fixes to the main branch, could you please pull the latest changes from main to fix the pipeline ?

zoro16 committed:
chore: Update README.md

makarono commented:
@zoro16 Thank you , I did pull the main branch.

zoro16 commented:
@makarono I think you need to pull/merge the changes into your `fix/lb-healthcheck` branch or rebase. 

Also I am not sure why does `zoro16:main` in the PR does not reflect the most recent changes ? any ideas ?

makarono commented:
@zoro16 I did that just now sorry, merged/rebased and pushed my fix/lb-healthcheck

zoro16 commented:
This PR is included in version 1.0.2 :tada:
